### ISSUE TITLE
implement else macro

### ISSUE THREAD

**Issue by amoibos** [haroldo-ok/twee2sam#26]
This would be nice and shouldn't be that hard.

**Comment by haroldo-ok** [comment]
I agree, though it would also require the implementation of an 'else' instruction on the SAM interpreter.

**Comment by amoibos** [comment]
My suggestion add a | symbol for the else branch within block. The start depending now of the expression. If it true skip all commands between | and ] otherwise ignoring all between [ and |. On my branch i already added the changes for the parser.
